applications/PoromechanicsApplication/python_scripts/poromechanics_MPI_U_Pw_solver.py

```diff
@@ -1,94 +1,32 @@
 from __future__ import print_function, absolute_import, division # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
-#import kratos core and applications
+
+# Importing the Kratos Library
 import KratosMultiphysics
-import KratosMultiphysics.mpi as mpi
+import KratosMultiphysics.mpi as KratosMPI
+
+# Check that applications were imported in the main script
+KratosMultiphysics.CheckForPreviousImport()
```

Comment:
I think this fct is currently not working, right @RiccardoRossi

Reply:
The recommended solution is to call KratosMultiphysics.CheckRegisteredApplications("...") instead.